### PR TITLE
Setup python package and Docker image publishing

### PR DESCRIPTION
**Pull request by pracucci**
In this PR I've done the setup to publish the python package and Docker image. I have not tested it yet, cause I would like to get a quick feedback on the overall changes. If they look good, I will publish a first version to test it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +54,33 @@ python3 -m unittest
 ```
 
 
+### How to publish a new version
```
Comment (esanchezm) [inline README.md]: I think this is more internal doc than something anyone would need on a README. If you really want it, move it in a Wiki page of the github repo.

File: README.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ Safety countermeasures:
 
 ## How to run it
 
+You have two options to run it:
+
+1. Manually install and run the [`aws-cloud-unmap` Python package](https://pypi.org/project/aws-cloud-unmap/)
+2. Use the [Docker image available on Docker hub](https://hub.docker.com/u/spreaker/aws-cloud-unmap/)
+
```
Comment (esanchezm) [inline README.md:37]: I think it's easier to include the exact commands to install and run for the first time. That's something I really appreciate when reading a README

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  description      = 'Remove terminated EC2 instances from AWS CloudMap service',
+  long_description = long_description,
+  author           = 'Marco Pracucci',
+  author_email     = 'marco@pracucci.com',
+  url              = 'https://github.com/spreaker/aws-cloud-unmap',
+  download_url     = 'https://github.com/spreaker/aws-cloud-unmap/archive/1.0.0.tar.gz',
+  keywords         = ['aws', 'cloud map'],
+  classifiers      = [],
+  python_requires  = ' >= 3',
+  install_requires = ['boto3==1.9.123', 'python-json-logger==0.1.10'],
```
Comment (esanchezm) [inline setup.py]: You could put the content of `requirements.txt`  here so you only mantain one file.

**Review by esanchezm — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pracucci) [reply README.md]: I would prefer to be able to manage it through the code editor. What do you think about `PUBLISH.md`?

File: setup.py (hunk 3 above)
Comment (pracucci) [reply setup.py]: What's about dev/test dependencies in `requirements.txt`? May you give me an example, please?

File: README.md (hunk 1 above)
Comment (esanchezm) [reply README.md]: Sure, is way better :)

File: README.md (hunk 2 above)
Comment (pracucci) [reply README.md:37]: How does it look, now?

File: setup.py (hunk 3 above)
Comment (esanchezm) [reply setup.py]: Normally you would have a `requirements.txt` per environment. But one thing you could is keep this as is for releases and have another `requirements-dev.txt` like this:

```
-r requirements.xt
dev_req==1.0.0
```

File: README.md (hunk 2 above)
Comment (esanchezm) [reply README.md:37]: Way better! Just after the `pip3`, you should add a running example similar to the docker one.

File: setup.py [hunk 4]
```diff
@@ -21,7 +25,7 @@
   keywords         = ['aws', 'cloud map'],
   classifiers      = [],
   python_requires  = ' >= 3',
-  install_requires = ['boto3==1.9.123', 'python-json-logger==0.1.10'],
+  install_requires = [requirements],
```
Comment (esanchezm) [inline setup.py]: `requirements` is already a list

**Review by esanchezm — CHANGES_REQUESTED** [review]

File: setup.py (hunk 4 above)
Comment (esanchezm) [reply setup.py]: I'll approve because it's trivial

**Review by esanchezm — APPROVED** [review]

**Comment by esanchezm** [comment]
Nice work! 👍

**Comment by pracucci** [comment]
@esanchezm I did some changes since your approval and I would like a re-review:
1. Upload package with `twine`
2. Moved dependencies to `setup.py` and load them from `setup.py` in `requirements.txt` (otherwise we need to add `requirements.txt` to the package in order for the installation to work correctly)
3. Added the function `run()` which is the one referenced in `entry_points` in `setup.py`

**Comment by esanchezm** [comment]
I see you find a different approach for the `requirements.txt`. Great!

**Review by esanchezm — APPROVED** [review]

**Pull request merged**